### ISSUE TITLE
Cannot Open URL

### ISSUE THREAD

**Issue by fzwo** [BurntCaramel/Lantern#5]
When trying to add a URL by hitting Cmd+L or using the Open URL main menu item, nothing happens.

Console output:
`Failed to connect (saveInFavorites) outlet from (Lantern.SiteSettingsViewController) to (NSButton): missing setter or instance variable`

This is on macOS 10.14.6, with Lantern 1.1.0 (18) from the App Store.

**Comment by nguyenhuuhanhch** [comment]
I have the sames issue. Please help me check and fix it.

**Comment by royalicing** [comment]
I’ve released version 1.1.1 which I think should fix the issue — does this work? @fzwo @nguyenhuuhanhch? Thanks

**Comment by nguyenhuuhanhch** [comment]
It’s still issues. I have post a issue in github.

**Comment by royalicing** [comment]
(https://github.com/RoyalIcing/Lantern/issues/6)

**Comment by fzwo** [comment]
Unfortunately, still doesn't work with latest App Store build (though thank you for the thanks!). No messages in Console anymore though.

**Comment by royalicing** [comment]
(Copying note from issue #6:)

I was able to delete my container (containing preferences etc) and replicate the issue. Version 1.1.3 is out on the Mac App Store and fixes it.

I have changed the toolbar to have a URL field instead of the button + popover. Let me know if there are any issues. Note — I removed the favorites system, which I felt made getting started slower and was more clunky than I wanted. I can focus on adding a new favorites system if anyone would like that (open another issue for this).
